hle-survey/transformation/src/main/scala/org/broadinstitute/monster/dap/healthcondition/HealthConditionType.scala

```diff
@@ -0,0 +1,44 @@
+package org.broadinstitute.monster.dap.healthcondition
+
+import enumeratum.values.{LongEnum, LongEnumEntry}
+
+sealed abstract class HealthConditionType(
+  override val value: Long,
+  val label: String,
+  val cgKey: Option[HealthConditionType.CgKey],
+  val dxKey: Option[HealthConditionType.DxKey]
```

Comment:
This information was previously set on a per-condition basis, but it was largely copy-pasted since it's really a property of each "block" of conditions.